ci/test.sh

```diff
@@ -17,8 +17,11 @@
 set -ex
 
 gcc -fPIC -shared -o ci/libnss_whatami.so.2 ci/libnss_whatami.c
-sudo cp ci/libnss_whatami.so.2 /lib
-sudo sed -i 's/\(passwd\|group\):/& whatami/' /etc/nsswitch.conf
-sudo dpkg -i nsncd*.deb
+cp ci/libnss_whatami.so.2 /lib
+sed -i 's/\(passwd\|group\):/& whatami/' /etc/nsswitch.conf
+dpkg -i nsncd*.deb
+/usr/lib/nsncd &
```

Comment:
If you need to do this it means the `dpkg -i` isn't starting the daemon correctly (possibly because you're in a container without systemd? less likely, possibly because there's a /usr/sbin/policy-rc.d file in whatever chroot/environment you're in?). I think we should figure out how to run the test in a context where systemd is running, if possible, so we have test coverage for the systemd unit. Maybe that involves splitting this file into two separate tests, one for the systemd unit on the uncontained ubuntu-latest runner and one for the various platforms we support in containers.

Reply:
It was because there's no systemd in the container, I tried running systemctl status and the command wasn't there. 

Yeah, I can add a test on Ubuntu with systemd